includes/html/print-interface.inc.php

```diff
@@ -72,6 +72,10 @@
 
 echo '</span>';
 
+$port_group_name_list = dbFetchColumn('SELECT port_groups.name FROM port_groups LEFT JOIN port_group_port ON port_groups.id=port_group_port.port_group_id WHERE port_group_port.port_id=?', [$port['port_id']]) ?: ['Default'];
```

Comment:
why not use eloquent?

Reply:
so whole page would be classical style and a single eloquent unicorn ;-)

Reply:
The classical sql queries are the unicorns.  Blocking our way to alternative SQL dialect support ;)

Reply:
updated :-)